operate/access-control.mdx

```diff
@@ -17,6 +17,15 @@ For database privileges that can be applied to each of the object levels, see [P
 
 ## Users
 
+RisingWave automatically creates three superuser accounts during cluster initialization:
+- `root` - Default superuser account
+- `postgres` - PostgreSQL-compatible superuser account  
+- `rwadmin` - Reserved superuser account for cloud control plane operations
+
+<Note>
+The `rwadmin` user cannot be altered, renamed, or dropped to ensure system security. Only the `rwadmin` user itself can change its password. For existing clusters, `rwadmin` is not automatically created but can be manually created by other superusers if needed.
```

Comment:
```suggestion
The `rwadmin` user cannot be altered, renamed, or dropped to ensure system security. Only the `rwadmin` user itself can change its password.
```
This part is only needed when the cloud internal upgrades the version. No need to make it public in doc.